modules/virt-example-nmstate-IP-management.adoc

```diff
@@ -60,12 +60,14 @@ The following snippet configures an Ethernet interface that uses a dynamic IP ad
 ...
```

Comment:
Please keep the dynamic example as it was, and add a static example as well.
In the static example - please keep `enabled` under `ipv4` and not under `address`.
Thank you!

Reply:
@yossisegev - To summarize your recommendations, is this what we want for this contet:
Examples: IP management
  Static
  No IP address
  Dynamic host configuration
  Static host configuration
  DNS
  Static routing

Reply:
Retained the content for dynamic example. Added the static example.